### PR TITLE
Release 2.9.0

### PR DESCRIPTION
**Pull request by benib**
Changes from the feature branch:

- renamed projection to error margin throughout the code to be more generic
- added two inputs for the error margin legend labels (to overwrite "Hochrechnung" with something else if it makes sense)

- deployed on stage

### CODE REVIEW

File: views/Footer.html [hunk 1]
```diff
@@ -45,10 +45,11 @@
           }
         }
       },
-      shouldHaveUpdateInfo: (updatedDate, hideUpdatedDate) => {
+      shouldHaveUpdateInfo: ({ updatedDate, hideUpdatedDate }) => {
+        debugger;
```
Comment (manuelroth) [inline views/Footer.html]: This should be removed

Comment (benib) [reply views/Footer.html]: you are right, it's removed now

File: resources/fixtures/data/results-threshold-color-codes-uncertainty.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  "withErrorMargin": true,
+  "errorMarginLabels": {
+    "bestGuessLabel": "Best guess",
+    "errorMarginLabel": "Fehlermarge"
+  },
+  "threshold": 5,
+  "createdDate": "2019-03-12T07:30:52.700Z",
+  "createdBy": "anna.wiederkehr@nzz.ch",
+  "updatedDate": "2019-03-12T07:33:13.235Z",
+  "updatedBy": "anna.wiederkehr@nzz.ch"
```
Comment (manuelroth) [inline resources/fixtures/data/results-threshold-color-codes-uncertainty.json]: Could you remove unnecessary properties createdDate, createdBy, updatedDate, updatedBy, publication, acronym and department?

File: resources/locales/en/translation.json [hunk 3]
```diff
@@ -6,12 +6,18 @@
   "Der Untertitel benennt präzise, um welche Wahl es sich handelt&#46;": "the subtitle gives a precise name of what election is concerned&#46;",
   "% Hürde": "threshold in %",
   "Falls eine Prozenthürde exisitert, bitte Zahl eintragen&#46;": "if a percentage hurdle exists, please enter the number&#46;",
+  "Daten mit Fehlermarge": "Data with error margin",
```
Comment (manuelroth) [inline resources/locales/en/translation.json]: String "mit Fehlerbereich" is missing

File: views/HtmlStatic.html [hunk 4]
```diff
@@ -1,41 +1,79 @@
...
+          return party.errorMargin !== undefined && party.errorMargin.bestGuess !== undefined;
+        });
+        return currentResults.length > 0;
+      },
+      sortedParties: ({ parties, item }) => {
         if (parties !== undefined) {
+          if (item.withErrorMargin) {
+            return parties.sort(function(partyA, partyB) {
+              // if any of the 2 parties has no errorMargin, we sort by this first
+              if (!partyA.errorMargin && !partyB.errorMargin) {
```
Comment (manuelroth) [inline views/HtmlStatic.html]: Should be sorted based on bestGuess as well. See ->

Bildschirmfoto 2019-03-20 um 16 27 58

File: resources/locales/en/translation.json (hunk 3 above)
Comment (benib) [reply resources/locales/en/translation.json]: yes

File: views/HtmlStatic.html (hunk 4 above)
Comment (benib) [reply views/HtmlStatic.html]: this gets sorted by bestGuess only now.

File: resources/locales/en/translation.json [hunk 5]
```diff
@@ -6,12 +6,19 @@
...
+  "Beschriftung der Legende": "Legend labels",
+  "Beschriftung des besten Werts in der Legende": "Best guess label",
+  "Beschriftung des Fehlerbereichs": "Error margin label",
   "Parteien": "parties",
   "Partei": "party",
   "Parteiname": "party name",
   "Parteifarbe": "party color",
   "Farbcode": "color code",
   "CSS Klassenname": "CSS class name",
+  "mit Fehlerbereich": "with error margin",
```
Comment (manuelroth) [inline resources/locales/en/translation.json]: I think we should consistently use "Fehlerbereich" or "Fehlermarge" and not use both

Comment (benib) [reply resources/locales/en/translation.json]: done

**Pull request merged**